Emby.Server.Implementations/Library/Resolvers/Movies/MovieResolver.cs

```diff
@@ -387,7 +387,7 @@ private static void SetProviderIdsFromPath(Video item)
 
                 if (!string.IsNullOrEmpty(item.Path))
                 {
-                    // check for imdb id - we use full media path, as we can assume, that this will match in any use case (either id in parent dir or in file name)
+                    // Check for IMDb id - we use full media path, as we can assume that this will match in any use case (wither id in parent dir or in file name)
```

Comment:
```suggestion
                    // Check for IMDb id - we use full media path, as we can assume that this will match in any use case (whether id in parent dir or in file name)
```